backend/reminders/budgetReminder.js

```diff
@@ -0,0 +1,51 @@
+const { startOfMonth, endOfMonth } = require('date-fns');
```

Comment:
```suggestion
const { startOfWeek, endOfWeek } = require('date-fns');
```

do you import the right functions?

Reply:
I'd initially started with the plan to do monthly budgets but made the switch to weekly and must have missed this, thank you!